src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,116 @@
+import clsx from 'clsx';
+import React, {
+  useId,
+  type ChangeEventHandler,
+  type CSSProperties,
+} from 'react';
+import styles from './Slider.module.css';
+import Label from '../Label';
+
+export type Props = {
+  /**
+   * Aria-label to provide an accesible name for the text input if no visible label is provided.
+   */
+  'aria-label'?: string;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Disables the slider and prevents change.
+   */
+  disabled?: boolean;
+  /**
+   * HTML id for the component
+   */
+  id?: string;
+  /**
+   * HTML label text
+   */
+  label?: string;
+  /**
+   * List of markers to imply slider value.
+   */
+  markers?: string[];
```

Comment:
Currently constrained to strings (which could also incl emoji, for better or for worse) and evenly separates them out via `justify-content: space-between`. Could consider expanding to `ReactNode` if the need arises

Reply:
I think strings should be enough for our usecase at least

Reply:
This is a little too open-ended at the moment. I think the intent is that the markers will be determined by the `min`/`max`/`step` values, and the number of markers is mathematically determined by those three props when the display some arbitrary text (`max - min / step`).

One way to account for this is to add checks to make sure markers is properly constrained. That would be useful whether they are simple strings or other symbols, or `ReactNode[]`. spec tests to verify it works too.

Also, if the common case is that markers are always these numeric sets, we can add a marker utility function to generate the steps. For example, `markers` is either `string[]` or `'numeric'` (a special documented keyword), where we check for `numeric` and generate the array values for consumers.

Reply:
> This is a little too open-ended at the moment. I think the intent is that the markers will be determined by the `min`/`max`/`step` values, and the number of markers is mathematically determined by those three props when the display some arbitrary text (`max - min / step`).

I like this idea

Another idea is that we could update markers to be customizable `markers?: Array<{ label?: string, value: number }>` instead of `markers: Array<string>`, but then placing the actual markers may be get complicated

Reply:
Yea placing the markers according to the value of the marker (like how `<option>` is done) does complicate things and make the prop a little more complex than I want. However, if the need arises for it in a product, will definitely be considered.

Andrew and I did pair on the generation of the markers which is now in this pr